pkg/collector/corechecks/cluster/orchestrator/pod.go

```diff
@@ -16,6 +16,7 @@ import (
 	"time"
 
 	model "github.com/DataDog/agent-payload/process"
+	"github.com/DataDog/datadog-agent/pkg/orchestrator"
```

Comment:
I moved the `pod.go` file because it was importing `	"github.com/DataDog/datadog-agent/pkg/orchestrator/config"` which was importing `pkg/forwarder` which now imports `pkg/orchestrator`.

Reply:
Could we solve this another way?  (new subpackage?)
I find it a bit misleading to have this under `collector/corechecks` because this one is a process check and not a core check

Reply:
Yes, indeed. I can move this around.

But I think it is part of both: DCA check + Process check (as the code only works around pod processing)

Reply:
I decided to move the `pods` part back but move the `forwarder` from `orchestrator.config.NewOrchestratorF 663790d4b9c53c0e735c60c3f3b04111386db927.
Thinks compile. Test locally as well, I will wait for online and integration test to run

Reply:
Locally things still work:
`process-agent`

```
2021-08-09 16:30:28 UTC | PROCESS | INFO | (collector.go:159 in runCheck) | Finished pod check #1 in 1.187450134s
2021-08-09 16:30:29 UTC | PROCESS | INFO | (pkg/forwarder/transaction/transaction.go:365 in internalProcess) | Successfully posted payload to "https://orchestrator.datadoghq.com/api/v1/orchestrator", the agent will only log transaction success every 500 transactions
2021-08-09 16:30:37 UTC | PROCESS | INFO | (collector.go:159 in runCheck) | Finished process check #2 in 65.192579ms
2021-08-09 16:30:38 UTC | PROCESS | INFO | (collector.go:159 in runCheck) | Finished pod check #2 in 3.319946ms
```

`DCA`
```
2021-08-09 16:27:02 UTC | CLUSTER | INFO | (pkg/collector/runner/runner.go:269 in work) | check:orchestrator | Running check
2021-08-09 16:27:02 UTC | CLUSTER | INFO | (pkg/collector/runner/runner.go:337 in work) | check:orchestrator | Done running check, next runs will be logged every 500 runs
```


`Status Output` as well
```
=====================
Orchestrator Explorer
=====================
  Collection Status: The collection is at least partially running since the cache has been populated.
  Cluster Name: nammn
  Cluster ID: 9e99cefc-06cb-44a7-a0d5-69aaf0ff5a47

  ======================
  Orchestrator Endpoints
  ======================
  https://orchestrator.datadoghq.com - API Key ending with: f975c

  ===========
  Cache Stats
  ===========
    Elements in the cache: 27
    Pods:
      Last Run: (Hits: 1 Miss: 0) | Total: (Hits: 29 Miss: 2)
    Deployments:
      Last Run: (Hits: 2 Miss: 0) | Total: (Hits: 58 Miss: 4)
    ReplicaSets:
      Last Run: (Hits: 2 Miss: 0) | Total: (Hits: 58 Miss: 4)
    Services:
```